core/cc/archive.cpp

```diff
@@ -76,35 +221,32 @@ Archive::Archive(const std::string& archiveName) {
 }
 
 Archive::~Archive() {
-  if (mDataFile) fclose(mDataFile);
+  mDataFile.close();
   if (mIndexFile) fclose(mIndexFile);
 }
 
-bool Archive::contains(const std::string& id) const {
+bool Archive::contains(const std::string &id) const {
   return mRecords.find(id) != mRecords.end();
 }
 
-bool Archive::read(const std::string& id, void* buffer, uint32_t size) {
+bool Archive::read(const std::string &id, void *buffer, uint32_t size) {
```

Comment:
nit: I believe clang-format will position these based on the common usage in the file. We position them next to the type elsewhere in our codebase.

Reply:
Thanks.